### PR TITLE
Implement Apprise for push notifications

### PR DESCRIPTION
**Pull request by rklf**
Including:
- Better logging
- Apprise library implementation

Thanks to @HamletDuFromage for breaking the back of the work for me.

### CODE REVIEW

File: paybybot3/notifs.py [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
-    with SMTP("smtp.gmail.com", 587) as server:
-        server.starttls()
-        server.login(email, pwd)
-        email_text = EMAIL_TEMPLATE.format(
-            email=email, subject=subject, message=message
-        ).encode("utf8")
-        server.sendmail(email, to, email_text)
-        logging.info("sent email to %s with message: %s", email, message)
+def _send_notification(apobj, title, body, tag):
+    notify_type = TAG_MAPPING_NOTIFY_TYPE.get(tag[0]) if tag else None
```
Comment (HamletDuFromage) [inline paybybot3/notifs.py]: Unless I'm missing something, tag here (and in `notify`) doesn't need to be an array

**Review by HamletDuFromage — COMMENTED** [review]
I just found one array that could be a string. Aside from that, everything looks good to me

File: paybybot3/notifs.py (hunk 1 above)
Comment (rklf) [reply paybybot3/notifs.py]: Nice catch. I wanted it to be multiple-tags ready, but this is not useful in this use-case.

File: paybybot3/notifs.py [hunk 2]
```diff
@@ -24,5 +24,5 @@ def _create_apprise_object(services):
     return apobj
 
 def _send_notification(apobj, title, body, tag):
-    notify_type = TAG_MAPPING_NOTIFY_TYPE.get(tag[0]) if tag else None
+    notify_type = TAG_MAPPING_NOTIFY_TYPE.get(tag) if tag else None
```
Comment (HamletDuFromage) [inline paybybot3/notifs.py]: `notify_type = TAG_MAPPING_NOTIFY_TYPE.get(tag) if tag else None` could just be `notify_type = TAG_MAPPING_NOTIFY_TYPE.get(tag)`

**Review by HamletDuFromage — APPROVED** [review]

**Pull request merged**